lib/matplotlib/backends/backend_wx.py

```diff
@@ -1117,14 +1092,6 @@ def __init__(self, canvas, coordinates=True):
         # and/or dc.GetContentScaleFactor()
         self._retinaFix = 'wxMac' in wx.PlatformInfo
 
-    prevZoomRect = _api.deprecate_privatize_attribute("3.3")
-    retinaFix = _api.deprecate_privatize_attribute("3.3")
-    savedRetinaImage = _api.deprecate_privatize_attribute("3.3")
-    wxoverlay = _api.deprecate_privatize_attribute("3.3")
-    zoomAxes = _api.deprecate_privatize_attribute("3.3")
-    zoomStartX = _api.deprecate_privatize_attribute("3.3")
-    zoomStartY = _api.deprecate_privatize_attribute("3.3")
```

Comment:
with these removals, the `press_zoom` and `release_zoom` overrides below have become noops and can be completely deleted, as well as the definition of _retinaFix just above.